src/unix/notbsd/linux/mod.rs

```diff
@@ -165,6 +165,20 @@ s! {
         _pad: [::uint8_t; 48],
     }
 
+
+    pub struct sigevent {
+        pub sigev_value: sigval_t,
+        pub sigev_signo: ::c_int,
+        pub sigev_notify: ::c_int,
+        pub sigev_notify_function: ::dox::Option<extern fn(sigev_value: sigval_t)>,
```

Comment:
looks like this should be a pointer to extern fn?

https://github.com/torvalds/linux/blob/v4.3/include/uapi/asm-generic/siginfo.h#L294